src/styles/blocks/card.scss

```
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  padding: 32px 16px 16px;
+  width: 200px;
+  background-color: $white-color;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  margin: 0 48px 46px 0;
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+  &__image {
```

```suggestion
  margin: 0 48px 46px 0;

  &:nth-child(4n) {
    margin-right: 0;
  }

  &__image {
```
add empty lines between selectors to increase readability